src/utils.rs

```diff
@@ -0,0 +1,48 @@
+use once_cell::sync::Lazy;
+
+use crate::errors::ApiError;
+
+
+pub static SECRET_KEY: Lazy<String> =
```

Comment:
It might be best to fail if SECRET_KEY is absent.

Reply:
Repair later